packages/SwingSet/test/test-xsnap-store.js

```diff
@@ -6,7 +6,8 @@ import fs from 'fs';
 import { tmpName } from 'tmp';
 import { type as osType } from 'os';
 import sqlite3 from 'better-sqlite3';
-import test from 'ava';
+// eslint-disable-next-line import/order
+import { test } from '../tools/prepare-test-env-ava.js';
```

Comment:
please avoid introducing a lint suppression.

Reply:
Aside from fixing https://github.com/endojs/endo/issues/1467 , how? See previous discussion at https://github.com/endojs/endo/pull/1439#discussion_r1070336130

Reply:
Yes, that seems to be the only way.  Out of scope, I suppose.

At the least please use `// eslint-disable-line` so that reordering imports doesn't break the association of the comment with the line.
```suggestion
import { test } from '../tools/prepare-test-env-ava.js'; // eslint-disable-line import/order
```